libraries/chain/include/eosio/chain/block_header_state_utils.hpp

```diff
@@ -20,6 +20,26 @@ namespace eosio::chain::detail {
       return block_timestamp_type{t.slot + (config::producer_repetitions - index) + config::producer_repetitions};
    }
```

Comment:
Remove unused `get_next_next_round_block_time`. The log statement using should not report when it will change because it can't know when that will be.

Reply:
https://github.com/AntelopeIO/spring/pull/483/commits/fa43246df35fe1e08d47f782bb27e66d86f950ac